mypy_django_plugin/main.py

```diff
@@ -91,6 +92,7 @@ class NewSemanalDjangoPlugin(Plugin):
     def __init__(self, options: Options) -> None:
         super().__init__(options)
         django_settings_module = extract_django_settings_module(options.config_file)
+        sys.path.extend(options.mypy_path)
```

Comment:
What about `MYPYPATH` env var?

Reply:
Good point. It wasn't looking there but I've updated it so that it will. I haven't included a test for the env var though. Is there a good way of setting env vars in the test suite?

I was looking at https://github.com/python/mypy/blob/master/mypy/modulefinder.py#L541 for inspiration. Is there anything else we should incorporate from there?